include/oneapi/dpl/experimental/kt/esimd_radix_sort_utils.h

```diff
@@ -436,6 +436,100 @@ VectorStore(uint32_t offset,
+
+
+    // vals, double, 8 bytes:
+    // +----------------------------------------------------------+-----------------------------------------------------------+-----+----------------------------------------------------------+
+    // |                             d0                           |                             d1                            | ... |                             dN                           |
+    // +----------------------------------------------------------+-----------------------------------------------------------+-----+----------------------------------------------------------+
+    //                                                                                          |
+    // __uint32_vals, uint32_t, 4 bytes:                                                        V
+    // +-----------------------------+----------------------------+-----------------------------+-----------------------------+-----+-----------------------------+----------------------------+
+    // | uint32_t(d0, first 4 bytes) | uint32_t(d0, last 4 bytes) | uint32_t(d1, first 4 bytes) | uint32_t(d1, last 4 bytes)  | ... | uint32_t(dN, first 4 bytes) | uint32_t(dN, last 4 bytes) |
+    // +-----------------------------+----------------------------+-----------------------------+-----------------------------+-----+-----------------------------+----------------------------+
+
+    // mask, simd_mask<N>:
+    // +----------------------------------------------------------+-----------------------------------------------------------+-----+----------------------------------------------------------+
+    // |                             m0                           |                             m1                            | ... |                             mN                           |
+    // +----------------------------------------------------------+-----------------------------------------------------------+-----+----------------------------------------------------------+
+    //                                                                                          |
+    // __uint32_mask, simd_mask<2 * N>:                                                         V
+    // +-----------------------------+----------------------------+-----------------------------+-----------------------------+-----+-----------------------------+----------------------------+
+    // |             m0              |             m0             |             m1              |             m1              | ... |             mN              |             mN             |
+    // +-----------------------------+----------------------------+-----------------------------+-----------------------------+-----+-----------------------------+----------------------------+
+
+    using TypeX32 = decltype(DestTypeResolver(T{}));
```

Comment:
Can you use just `using TypeX32 = ::std::uint32_t`?

Reply:
Fixed.